### ISSUE TITLE
[BUG] Installation error on dev setup

### ISSUE THREAD

**Issue by kayrocesar** [houdiniproject/houdini#1425]
## Description

Hello, I am a software engineering student at the University of Brasilia and I am trying to set up the development environment to contribute to the houdini project, but I am getting an error installing ruby:

Error running '__rvm_make install',

## Expected Behavior

Install ruby with the command:

'rvm install 2.7.6 --disable-binary --with-jemalloc'
## Actual Behavior

I follow the installation procedure as described, but when I run the command:

'rvm install 2.7.6 --disable-binary --with-jemalloc'.

I get the following message:

" Error running '__rvm_make install' " 

![Captura de tela de 2022-12-05 22-11-41](https://user-images.githubusercontent.com/39713656/205786640-8c522c7c-587a-4ec5-8711-51559ed1c055.png)

install.log :
![Captura de tela de 2022-12-05 22-37-55](https://user-images.githubusercontent.com/39713656/205788144-6185bc2b-dc2e-4815-b93d-86cc425847b1.png)

## Steps to Reproduce

All the steps from dev setup up to the command

rvm install 2.7.6 --disable-binary --with-jemalloc

## Your Environment

* Operating System and version: Ubuntu 22.04.1 LTS

**Comment by wwahammy** [comment]
@kayrocesar thanks for catching this! I actually noticed this bug on a private repo but didn't move it out here.

Can you replace the `rvm install 2.7.6 --disable-binary --with-jemalloc` in your install with the following lines:

```bash
rvm pkg install openssl
rvm install 2.7.6 --disable-binary --with-jemalloc --with-openssl-dir=$HOME/.rvm/usr
```

Does that work?

**Comment by wwahammy** [comment]
@kayrocesar I think I have a fix but it will require removing rvm and using rbenv (which seems better supported). I'll document that later tonight hopefully

**Comment by wwahammy** [comment]
@kayrocesar so I'm declaring "bankruptcy" on using rvm as I couldn't get it working when I started from scratch no matter what I did.  I was able to get rbenv working. Please run the following commands:

```bash
rm -rf ~/.rvm # assuming you don't want rvm any more

# manually go into your .bashrc and .bash_profile files and remove any lines related to rvm (should be at the bottom)

git clone https://github.com/rbenv/rbenv.git ~/.rbenv # get the latest rbenv

echo 'eval "$(~/.rbenv/bin/rbenv init - bash)"' >> ~/.bashrc # add rbenv to bashrc

# close and reopen your your terminal

git clone https://github.com/rbenv/ruby-build.git "$(rbenv root)"/plugins/ruby-build # download the rbenv install feature

cd ~/houdini # or wherever your houdini checkout is

rbenv install # this should install Ruby 2.7.6

bin/setup # run bin/setup and continue the installation

```

Please let me know if this works.

I have no clue what changed but rvm is clearly not working for builds any more.

**Comment by kayrocesar** [comment]
Hi @wwahammy , I did everything as requested and it worked !! I really appreciate your help. Build done!

Just an observation, in ubuntu 20.04 I could configure the development environment without any problem with RVM.

But on Ubuntu 22.04 and linux Mint 21 (based on ubuntu 22.04) I was having these problems with installing Ruby with Rvm and that error with the database when running bin/setup.

Do you want to replace RVM for RBENV in the README or do you want to comment on the problem with the ubuntu version? I would like to contribute with this in an improvement of the README through a Pull Request following what you put here in the issue. I say that, because some people I know had the same problem that I reported in this issue.

**Comment by joaobisi** [comment]
> @kayrocesar so I'm declaring "bankruptcy" on using rvm as I couldn't get it working when I started from scratch no matter what I did. I was able to get rbenv working. Please run the following commands:
> 
> ```shell
> rm -rf ~/.rvm # assuming you don't want rvm any more
> 
> # manually go into your .bashrc and .bash_profile files and remove any lines related to rvm (should be at the bottom)
> 
> git clone https://github.com/rbenv/rbenv.git ~/.rbenv # get the latest rbenv
> 
> echo 'eval "$(~/.rbenv/bin/rbenv init - bash)"' >> ~/.bashrc # add rbenv to bashrc
> 
> # close and reopen your your terminal
> 
> git clone https://github.com/rbenv/ruby-build.git "$(rbenv root)"/plugins/ruby-build # download the rbenv install feature
> 
> cd ~/houdini # or wherever your houdini checkout is
> 
> rbenv install # this should install Ruby 2.7.6
> 
> bin/setup # run bin/setup and continue the installation
> ```
> 
> Please let me know if this works.
> 
> I have no clue what changed but rvm is clearly not working for builds any more.

In addition to these steps, it's also recommended to check .zshrc and .profile files to delete rvm source lines in case of per-user installation.
I had the same issue and running the commands as you said solved it. thanks!

**Comment by wwahammy** [comment]
@kayrocesar I'm glad it worked! If the rbenv instructions also work on Ubuntu 20.04, then we would welcome a PR replacing rvm with rbenv in the README!

**Comment by wwahammy** [comment]
Closed by #1430
